src/resources/app/pages/login.vue

```diff
@@ -97,6 +97,7 @@ export default {
         }
       }
     },
+    submit() { },
```

Comment:
what is this

Reply:
was getting error because there's no `submit` function as written here https://github.com/uasoft-indonesia/badaso-lms-theme/pull/18#discussion_r825253903